### PR TITLE
Typos from GEANT/CAT

### PR DESCRIPTION
**Pull request by DimitriPapadopoulos**
From https://github.com/GEANT/CAT/pull/252.

### CODE REVIEW

**Comment by skangas** [comment]
LGTM.

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -21389,6 +21389,8 @@ meeds->needs
 meens->means
 meerkrat->meerkat
 meerly->merely
+meethod->method
```
Comment (jdufresne) [inline codespell_lib/data/dictionary.txt:21392]: maybe:

meethodology->methodology

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11077,7 +11077,7 @@ deviceremoveable->deviceremovable
 devicesr->devices
 devicess->devices
 devicest->devices
-devide->divide
+devide->divide, device,
```
Comment (jdufresne) [inline codespell_lib/data/dictionary.txt:11080]: maybe devides->devices too.

**Review by jdufresne — APPROVED** [review]

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:11089]: DOne

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:21405]: Found a few hits in GitHub indeed. Done.

**Review by luzpaz — APPROVED** [review]

**Pull request merged**